### PR TITLE
Rotate 45 equipments

### PR DESCRIPTION
**Pull request by rum-yasuhiro**
Rotate 45 degree all of the equipments of Mach-Zehnder interferometer for align to the slides on YouTube.
Screen Shot 2022-02-19 at 23 37 44

### CODE REVIEW

File: src/pages/Interferometry/MachZehnderCanvas.js [hunk 1]
```diff
@@ -1,111 +1,24 @@
...
-    // y-axis waves: inside sample
-    for (let posY = sample.y; posY < sample.y - sample.h; posY++) {
-      ctx.fillRect(
-        amplitude * Math.sin(waveNumberK * posY * refractionIndex) + sample.x,
-        posY,
-        lightBeamWidth,
-        lightBeamWidth,
-      );
-    }
+    const equipment = new MachZehnderEquipment(
```
Comment (zigen) [inline src/pages/Interferometry/MachZehnderCanvas.js]: we don't want to create this class in each animation frame.

Comment (rum-yasuhiro) [reply src/pages/Interferometry/MachZehnderCanvas.js]: How about this?
[0b9d36d](https://github.com/sfc-aqua/qleap-c2q-optics-js-app/pull/18/commits/0b9d36d79fe67faf2afc2b07b7780628247018f6)

File: src/pages/Interferometry/MachZehnderCanvas.js [hunk 2]
```diff
@@ -1,111 +1,21 @@
...
-const mirror1 = new DoubleRect(50, 150, 50, 5, 135, 1, 2);
-const mirror2 = new DoubleRect(250, 350, 50, 5, -45, 1, 2);
-
-const sample = new DoubleRect(50, 250, 50, -30, 0, 1, 0);
-
-function MachZehnderCanvas({ size: { width, height }, refractionIndex }) {
+function MachZehnderCanvas({ size: { width, height } }) {
   const cvs = useRef(null);
-
+  const equipment = new MachZehnderEquipment(
```
Comment (zigen) [inline src/pages/Interferometry/MachZehnderCanvas.js]: that's better, but I think it's a constant so we can move it to the outside of the function

Comment (rum-yasuhiro) [reply src/pages/Interferometry/MachZehnderCanvas.js]: Moved it!

**Review by zigen — APPROVED** [review]

**Pull request merged**